src/components/ChangelogDialog/components/Item/Item.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+import {block} from '../../../utils/cn';
+import {Label} from '../../../Label';
+import {Button} from '../../../Button';
+import {Icon} from '../../../Icon';
+import {PlayIcon} from '../../../icons/PlayIcon';
+import {Picture} from '../Picture/Picture';
+import {ChangelogItem, OnStoryClick} from '../../types';
+import i18n from '../../i18n';
+
+import './Item.scss';
+
+const b = block('changelog-dialog-item');
+
+export interface ItemProps {
+    className?: string;
+    data: ChangelogItem;
+    onStoryClick?: OnStoryClick;
+}
+
+export function Item({className, data, onStoryClick}: ItemProps) {
+    return (
+        <article className={b(null, className)}>
+            <div className={b('meta')}>
+                <div className={b('date')}>{data.date}</div>
```

Comment:
Can we use `<time>` here? This will require to change `data.date` to one of the [machine-readable formats](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time#valid_datetime_values), and format date here to required format.

```suggestion
                <time datetime={data.date} className={b('date')}>{getHumanDate(data.date)}</div>
```